### ISSUE TITLE
List of characters

### ISSUE THREAD

**Issue by ExcaliburZero** [Infinite-Dango-Studios/ganbatte#2]
This is a list of all of the characters in Ganbatte. If there are any that need to be added or changed, please reply to this issue and I will add/change them.

| Name | Description | Notes |
| :-- | :-- | :-- |
| Haruki | Main character and narrator | Protagonist |
| Kosuke Kasai | Haruki's best friend | Bachelor |
| Misaki Kasai | Kosuke's twin sister | Bachelorette |
| Mr. Kasai | Kouske's father |  |
| Ms. Kasai | Kouske's mother |  |
| Riku | The cafe boss |  |
| Sora | Cafe worker |  |
| Megumi | Cafe worker |  |
| Yuta | Cafe worker |  |
| Kokona Machida | Cafe worker, has cancer | Bachelorette |
| Yuuki Akuma | A satanic, yet kawaii chunnibyo girl | Bachelorette |
| Aiko Miki | An otaku who works at a host club | Bachelorette |
| Ren Nanami | An athletic lover | Bachelorette |
| Coach :id: | Ren Nanami's coach |  |
| Host Club Owner :id: | The owner of the host club that the Aiko Miki works at |  |
| Momoko Kawai | A bookish megane loli | Bachelorette |

:id: means that the character's name has not yet been decided on.

**Comment by juanitaylor** [comment]
alright! good to know! thanks!

**Comment by Sabre-Hecate** [comment]
Yuta was the other male worker, just looked it up.

**Comment by ExcaliburZero** [comment]
@Sabre-Hecate 
I have fixed Yuta on the list.

**Comment by ExcaliburZero** [comment]
I have added notes next to all of the characters on the list that will have story arcs.

**Comment by Sabre-Hecate** [comment]
Yay, thanks~!

Also, I have officially finalized names (because what else is 3:02 AM for???) so we can finally update the list for the most part. 

OK here are the names I added:
The host club otaku lover is "Aiko Miki" 
The athlete (who's a swimmer for fanservice reasons as highlighted by Jacoba) is named "Ren Nanami".
The Megane Loli is called "Momoko Kawai" YES, THAT IS A REAL LAST NAME I SWEAR I FOUND IT ON A JAPANESE SURNAMES SITE. 

Also, Kokona's surname is "Machida". 

So to summarize:
Aiko Miki
Ren Nanami
Momoko Kawai
Kokona Machida.

ALL THE DATEABLE PEOPLE ARE NAMED. YAY~!

**Comment by ExcaliburZero** [comment]
@Sabre-Hecate 
I have updated the list to use those names.

**Comment by Sabre-Hecate** [comment]
Yay!

OK, so when I introduce characters, does it have to be last name first? Like "Hi, I'm Nanami-Ren" and then does he have to call them, like, "Nanami-san" or "Nanami-chan" or the like until they become friends?

**Comment by ExcaliburZero** [comment]
@Sabre-Hecate 
You could really go either way with it. Last name first might be more authentic, but it may also be a bit more confusing.

**Comment by ExcaliburZero** [comment]
I think it would be best to move the character list to the resources repo rather than having an issue on it, since issues are meant to be for things that need to be done / fixed.

So I will make the list into a table file and add it to the resources repo.

**Comment by ExcaliburZero** [comment]
Moved character list to resources repo with Infinite-Dango-Studios/ganbatte-resources@62e0ab3d6f14b1ea9a05efa706e8f79fe79d9cb7 .

Since I have moved the character list, I will close this issue.
